src/cpp/benders/benders_core/BendersBase.cpp

```diff
@@ -251,6 +251,7 @@ void BendersBase::update_trace() {
   _trace[_data.it - 1]->_min_invest = std::make_shared<Point>(_data.min_invest);
   _trace[_data.it - 1]->_deleted_cut = _data.deletedcut;
   _trace[_data.it - 1]->_time = _data.timer_master;
+  _trace[_data.it - 1]->_subproblem_duration = _data.subproblem_timers;
   _trace[_data.it - 1]->_nbasis = _data.nbasis;
   _trace[_data.it - 1]->_invest_cost = _data.invest_cost;
   _trace[_data.it - 1]->_operational_cost = _data.subproblem_cost;
```

Comment:
Add reference to `_trace[_data.it - 1]` to improve readability. Something like `auto latest_trace& = _trace[_data.it - 1]`